lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -118,11 +123,19 @@ function applyConfigurationMiddlewares(newConfiguration, callback) {
  * @param {Function} next        Invokes the next middleware in the chain.
  */
 function handleCreateDeviceGroup(req, res, next) {
-    for (let i = 0; i < req.body.services.length; i++) {
-        req.body.services[i] = applyMap(apiToInternal, req.body.services[i]);
-        req.body.services[i].service = req.headers['fiware-service'];
-        req.body.services[i].subservice = req.headers['fiware-servicepath'];
-    }
+    if (req.body.configGroups) {
+        for (let i = 0; i < req.body.configGroups.length; i++) {
+            req.body.configGroups[i] = applyMap(apiToInternal, req.body.configGroups[i]);
+            req.body.configGroups[i].service = req.headers['fiware-service'];
+            req.body.configGroups[i].subservice = req.headers['fiware-servicepath'];
+        }
+    } else {    
+        for (let i = 0; i < req.body.services.length; i++) {
+            req.body.services[i] = applyMap(apiToInternal, req.body.services[i]);
+            req.body.services[i].service = req.headers['fiware-service'];
+            req.body.services[i].subservice = req.headers['fiware-servicepath'];
+        }
+   }
```

Comment:
The logic of different routes ( `/iot/services` and `/iot/configGroups` should be the same)

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11